sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2DataFrameSessionCatalogSuite.scala

```diff
@@ -144,13 +144,13 @@ private [connector] trait SessionCatalogTest[T <: Table, Catalog <: TestV2Sessio
   protected val catalogClassName: String = classOf[InMemoryTableSessionCatalog].getName
 
   before {
-    spark.conf.set(V2_SESSION_CATALOG.key, catalogClassName)
+    spark.conf.set(V2_SESSION_CATALOG_IMPLEMENTATION.key, catalogClassName)
   }
 
   override def afterEach(): Unit = {
     super.afterEach()
     catalog("session").asInstanceOf[Catalog].clearTables()
```

Comment:
This seems to be last remaining instance.
```scala
-    catalog("session").asInstanceOf[Catalog].clearTables()
+    catalog(SESSION_CATALOG_NAME).asInstanceOf[Catalog].clearTables()
```

Reply:
This test suite fails without this.

Reply:
Good catch!